gradle/versions.gradle

```diff
@@ -100,10 +100,11 @@ dependencyManagement {
 
     dependency 'tech.pegasys:jblst:0.3.3-1'
 
-    dependencySet(group: 'tech.pegasys.signers.internal', version: '2.0.0') {
+    dependencySet(group: 'tech.pegasys.signers.internal', version: 'develop') {
```

Comment:
Can you update this to a non-develop version of signers. Will need to do a release of signers first though.